src/hdmf/backends/io.py

```diff
@@ -43,6 +43,17 @@ def write(self, **kwargs):
         f_builder = self.__manager.build(container, source=self.__source)
         self.write_builder(f_builder, **kwargs)
 
+    @docval({'name': 'container', 'type': Container, 'doc': 'the Container object to write'},
+            {'name': 'exhaust_dci', 'type': bool,
+             'doc': 'exhaust DataChunkIterators one at a time. If False, exhaust them concurrently', 'default': True},
+            allow_extra=True)
+    def export(self, **kwargs):
```

Comment:
should add a docstring describing the purpose of the export function (and difference to calling write)